src/main/java/frc/robot/Robot.java

```diff
@@ -24,9 +29,43 @@ public class Robot extends TimedRobot {
      */
     @Override
     public void robotInit() {
+        // Record metadata
+        Logger.recordMetadata("ProjectName", BuildConstants.MAVEN_NAME);
+        Logger.recordMetadata("BuildDate", BuildConstants.BUILD_DATE);
+        Logger.recordMetadata("GitSHA", BuildConstants.GIT_SHA);
+        Logger.recordMetadata("GitDate", BuildConstants.GIT_DATE);
+        Logger.recordMetadata("GitBranch", BuildConstants.GIT_BRANCH);
+        switch (BuildConstants.DIRTY) {
+            case 0:
+                Logger.recordMetadata("GitDirty", "All changes committed");
+                break;
+            case 1:
+                Logger.recordMetadata("GitDirty", "Uncommitted changes");
+                break;
+            default:
+                Logger.recordMetadata("GitDirty", "Unknown");
+                break;
+        }
+
+        if (isReal()) {
+            Logger.addDataReceiver(new WPILOGWriter("/media/sda1")); // Log to a USB stick
+            Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
+            // new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
+            setUseTiming(true);
+        } else {
```

Comment:
Currently this disallows simulation altogether ([findReplayLog](https://github.com/Mechanical-Advantage/AdvantageKit/blob/5aa701f0eb649981a19b10a2417e7c0291fb06f2/junction/core/src/org/littletonrobotics/junction/LogFileUtil.java#L43) will halt until a valid input source is given if it can't find a replay log automatically). We should pull out the `findReplayLog` into a separate method and allow it to return null, then do null checks here.

Reply:
I added an additional check for simulation mode. Will that work??